sdk/src/wallet/utxo.ts

```diff
@@ -186,6 +194,13 @@ export function getInputFromUtxoAndTx(
         }
         const inner = p2wpkh(Buffer.from(publicKey!, 'hex'), getBtcNetwork(network));
         redeemScript = p2sh(inner);
+    } else if (addressType === AddressType.p2tr) {
+        if (!publicKey) {
```

Comment:
This check is already done above in line 81.

Reply:
Will remove